components/common/SwitchButton.vue

```diff
@@ -38,6 +40,15 @@ export default {
       return this.options[0].to ? 'nuxt-link' : 'button'
     },
   },
+  methods: {
+    formatLabel(option) {
+      if (this.type === 'button') {
+        return formatIANATimeZone(option)
+      } else {
+        return option
+      }
+    },
```

Comment:
💬 A small suggestion 😄 ! Whenever you have this pattern `if/else` with both returns you can skip the `else` like this:
```js
if (something) {
  return "Yay!";
}
return "Nope!";
```